core/src/jvmTest/kotlin/com/ustadmobile/core/contentformats/GlobalDisptacher.kt

```diff
@@ -0,0 +1,63 @@
+package com.ustadmobile.core.contentformats
+
+import okhttp3.mockwebserver.Dispatcher
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.RecordedRequest
+import okio.Buffer
+import okio.Okio
+import okio.buffer
+import okio.source
+import org.apache.commons.io.IOUtils
+import java.io.File
+import java.io.IOException
+import java.nio.file.Files
+
+
+val globalDisptacher: Dispatcher = object : Dispatcher() {
```

Comment:
This "global" dispatcher is used in only one test, and it is really the same concept as ContentDispatcher.

I think you need one class: ResourcesDispatcher. That could take in a base path argument e.g.

```
ResourcesDispatcher(private val resourceBasePath: String)

 fun dispatch(request) {
    val resPath = UMFileUtil.joinPaths(resourceBasePath, request.path)
  }
}
```